pkg/agent/multicluster/mc_route_controller.go

```diff
@@ -43,9 +51,17 @@ const (
 	minRetryDelay = 2 * time.Second
 	maxRetryDelay = 120 * time.Second
 
-	// Default number of workers processing a resource change
-	defaultWorkers = 1
-	workerItemKey  = "key"
+	workerItemKey = "key"
+
+	TrafficEncryptionModeWireGuard = "wireGuard"
+	MulticlusterWireGuardPort      = 51821
```

Comment:
Reuse `apis.WireGuardListenPort`?

Reply:
I have concerns about the port conflict, maybe the users will be confused when the in-cluster WG and cross-cluster WG use the same port?

Reply:
Do you mean functionally it should work if inter-cluster and intra-cluster share the same WG port, but you believe a better practice is to use different ports?

Reply:
Yes. When the users are using inter or cross-cluster WG, they can easily distinguish if the traffic is cross-cluster or not from `wg` command, which shows the port. It's OK to use the same port, but I think maybe different is better?